hexrd/ui/overlays/powder_diffraction.py

```diff
@@ -124,8 +124,16 @@ def generate_ring_points(self, tths, etas, panel, display_mode):
             # construct ideal angular coords
             ang_crds = np.vstack([np.tile(tth, len(etas)), etas]).T
 
-            # !!! must apply offset
-            xys_full = panel.angles_to_cart(ang_crds, tvec_c=self.tvec)
+            # Convert nominal powder angle coords to cartesian
+            # !!! Tricky business; here we must consider _both_ the SAMPLE
+            #     CS origin and anything specified for the XRD COM for the
+            #     overlay.  This is so they get properly mapped back to the
+            #     the proper cartesian coords.
+            xys_full = panel.angles_to_cart(
+                ang_crds,
+                tvec_s=self.instrument.tvec,
+                tvec_c=self.tvec
```

Comment:
This makes me think we need to fix `tvec` in other calls to `panel.angles_to_cart()` in hexrdgui. It looks to me that in several places, we have been using `instr.tvec` for `tvec_c`, whereas this looks to me like we should be using the powder options `tvec` and set `tvec_s` to the instrument `tvec.